### PR TITLE
AMPQ added and configured, ApproveEvent added

### PR DESCRIPTION
**Pull request by SerjKocherga**

### CODE REVIEW

File: CharlieBackend.Api/AddEasyNetQExtension.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        //public static void UseEasyNetQ(this IApplicationBuilder app)
+        //{
+        //    var bus = app.ApplicationServices.GetRequiredService<IBus>();
+        //    var autoSubscriber = new AutoSubscriber(bus, "productor")
+        //    {
+        //        AutoSubscriberMessageDispatcher = app.ApplicationServices.GetRequiredService<IAutoSubscriberMessageDispatcher>()
+        //    };
+        //    autoSubscriber.Subscribe(Assembly.GetExecutingAssembly());
+        //    autoSubscriber.SubscribeAsync(Assembly.GetExecutingAssembly());
+        //}
```
Comment (SnipeFrost) [inline CharlieBackend.Api/AddEasyNetQExtension.cs]: Do we need this code above?

File: CharlieBackend.Core/IntegrationEvents/Abstractions/IEvent.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿namespace CharlieBackend.Core.IntegrationEvents.Abstractions
+{
+    public interface IEvent
+    {
+    }
+}
```
Comment (SnipeFrost) [inline CharlieBackend.Core/IntegrationEvents/Abstractions/IEvent.cs]: enter in the end of file

**Review by SnipeFrost — APPROVED** [review]

**Review by BohdanBBG — APPROVED** [review]

**Review by sober607 — APPROVED** [review]

File: CharlieBackend.Business/Services/StudentService.cs [hunk 3]
```diff
@@ -16,15 +18,17 @@ public class StudentService : IStudentService
         private readonly IUnitOfWork _unitOfWork;
         private readonly ICredentialsSenderService _credentialSender;
         private readonly IMapper _mapper;
+        private readonly IBus _bus;
 
         public StudentService(IAccountService accountService, IUnitOfWork unitOfWork, 
                               ICredentialsSenderService credentialsSender,
-                              IMapper mapper)
+                              IMapper mapper, IBus bus)
```
Comment (whereismyroot) [inline CharlieBackend.Business/Services/StudentService.cs:25]: Please place each parameter on its own line

File: CharlieBackend.Business/Services/MentorService.cs [hunk 4]
```diff
@@ -17,14 +19,16 @@ public class MentorService : IMentorService
         private readonly IUnitOfWork _unitOfWork;
         private readonly ICredentialsSenderService _credentialsSender;
         private readonly IMapper _mapper;
+        private readonly IBus _bus;
 
         public MentorService(IAccountService accountService, IUnitOfWork unitOfWork, ICredentialsSenderService credentialsSender,
-                             IMapper mapper)
+                             IMapper mapper, IBus bus)
```
Comment (whereismyroot) [inline CharlieBackend.Business/Services/MentorService.cs:25]: Please place each parameter on its own line

**Review by whereismyroot — APPROVED** [review]
Minor comments

**Pull request merged**